com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -3048,6 +3026,7 @@ static void ClearLightList(in BuildGPULightListParameters parameters, CommandBuf
             cmd.SetComputeBufferParam(parameters.clearLightListCS, parameters.clearLightListKernel, HDShaderIDs._LightListToClear, bufferToClear);
             cmd.SetComputeIntParam(parameters.clearLightListCS, HDShaderIDs._LightListEntries, bufferToClear.count);
 
+            // TODO: Round on DeviceInfo.optimalThreadGroupSize so we have optimal thread for ClearList kernel
```

Comment:
In current shape of things, won't this give the wrong number of groups dispatched? We still set NR_THREADS to 32 in the shader if that's the optimal lane count.

Reply:
It's a comment that this needs to be updated once PLATFORM_LANE_COUNT_32 path is enabled in ClearLightLists.compute, it's currently commented out

Reply:
What I mean is that if anywhere PLATFORM_LANE_COUNT is defined != 64 this will fail.

(This is in the shader) 

#ifdef PLATFORM_LANE_COUNT
#define NR_THREADS              PLATFORM_LANE_COUNT
#else
#define NR_THREADS              64                                  // default to 64 threads per group
#endif